source/getting-started/control-with-amdc/encoder-fb/index.md

```diff
@@ -1,13 +1,97 @@
 # Encoder Feedback
 
+## Background
+
+Encoders provide rotor position feedback to the control system in a motor drive. This document describes a method to convert the raw readings of an encoder into meaningful position information which can be used by the contrl algorithm. Consequently, few techniques to derive the rotor speed from the measured position are also proposed. For more information on how an encoder works and how they may be interfaced with the AMDC please refer to this [document](https://docs.amdc.dev/hardware/subsystems/encoder.html#).
```

Comment:
> Consequently, few techniques to derive the rotor speed from the measured position are also proposed.

This is a confusing sentence. It could mean that only a few techniques exist in the world. Or that we are proposing techniques here for the first time ever. 

Do you need the sentence -- are you trying to convey information that is not already in the previous sentence?